src/panel/applets/tray/TrayApplet.vala

```diff
@@ -172,7 +172,8 @@ public class TrayApplet : Budgie.Applet {
 		if (key in items) return;
 
 		try {
-			var new_item = new TrayItem(name, object_path, panel_size);
+			var size = settings.get_boolean("scaling") ? panel_size : 36;
```

Comment:
Given this is duplicated on line 82 during the construction of the TrayApplet, I think we should have this be refactored to a function, and have the `36` be defined as a public const that we can more easily change in one place.

I think there is likely further room for improvement in this area that is out-of-scope of this PR, but maybe worth keeping in mind for the future. I would probably have the panel default size (not user set size, e.g. default panel size defined is 36 already) be used as the basis for the value instead, however that value isn't exposed and we only are informed of panel size changes, so updating applet or applet info to have the associated panel's UUID and make it easier to query those default values would be nice. We're already 99% of the way there in panel.vala when it comes to our from_uuid constructor calls, we'd just want to pass the panel's uuid and have a function. Or maybe we can do a cast of the applet's parent window to a Panel (as Panel inherits from Toplevel, which itself inherits from Gtk.Window) and have a getter live in the panel :shrug: